__tests__/__output/json-nested.json

```diff
@@ -0,0 +1,12 @@
+{
+  "color": {
+    "base": {
+      "red": {
+        "primary": "#611D1C",
+        "secondary": {
+          "inverse": "#000000"
+        }
+      }
+    }
+  }
+}
```

Comment:
I'm not sure where this came from... Is this ok @dbanksdesign ?

Reply:
I think there might be a race condition in our unit tests somewhere.. there shouldn't be anything checked into **__tests__/__output/** that is the folder where we put files generated by the unit tests. It should get cleared after tests run. You can remove this file or just try running the tests again and see if it gets removed.